internal/liqonet/route-operator/symmetricRoutingOperator.go

```diff
@@ -148,14 +149,13 @@ func (src *SymmetricRoutingController) podFilter(obj client.Object) bool {
 		klog.Infof("object {%s} is not of type corev1.Pod", obj.GetName())
 		return false
 	}
-	// Check if pod is running on same node as the operator.
-	if p.Spec.NodeName == src.nodeName {
-		klog.V(4).Infof("skipping pod {%s} running on our same node {%s}", p.Name, p.Spec.NodeName)
-		return false
-	}
 	// If podIP is not set return false.
+	// Here the newly created pods scheduled on a virtual node will be skipped. The filtered cache for all the pods
+	// scheduled on a virtual node works only when the correct label has been added to the pod. When pods are created
+	// the label is not present, but we are sure that it will be added before the IP address for the same pod is set.
+	//Once the pods have been labeled the api server should not inform the controller about them.
```

Comment:
```suggestion
	// Once the pods have been labeled the api server should not inform the controller about them.
```

Unclear to me: `should not` or `should`?